ProcessList.c

```diff
@@ -162,15 +197,18 @@ static void ProcessList_updateTreeSetLayer(ProcessList* this, Hashtable* newTree
 
       if (indent == -1) {
          proc->indent = 0;
-      } else if (i == size - 1)
+      } else if (i == size - 1) {
          proc->indent = -currentIndent;
-      else
+      }
+      else {
```

Comment:
Should be only one line:

```suggestion
      } else {
```